### DISCUSSION TITLE
Risk Scenarios

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#98] [Q&A]
One huge centralisation issue is that the secretdecks.com domain is payed by, accounted to, held and managed by one individual (@tamagit80) we need to figure out how to incorporate it into the distributed cooperative (DisCo).

First thing that comes to mind is setup an Otoco LLC. 
But we could also go for a Verein (as  discussed here #65) .
**Now is the time to decide.** 

We could go for a _fast and inexpensive_ (Verein) solution or we can use the _more advanced, expensive & powerful_ (GmbH with golden share) solution.

@wlstylr @stephen-rowan How should we decide on how to decide? When voting does everyone has the same voting power?

**Comment by tamagit80** [comment]
This is a problem and same with the ownership of the discord server,
newsletter and git. We need to create an "entity" and move all the
ownership to it asap.

**Comment by tamagit80** [comment]
Also the wallet...

**Reply by wlstylr** [reply]
The wallet is a tough problem to solve at the moment.  At the least, we want a multisig wallet.  Ideally this would evolve into something where fund management is managed through the governance layer and automatically executed after passing vote.

**Comment by wlstylr** [comment]
> One huge centralisation issue is that the secretdecks.com domain is payed by, accounted to, held and managed by one individual (@tamagit80) we need to figure out how to incorporate it into the distributed cooperative (DisCo).
> 
> First thing that comes to mind is setup an Otoco LLC.
> But we could also go for a Verein (as discussed here #65) .
> **Now is the time to decide.**
> 
> We could go for a _fast and inexpensive_ (Verein) solution or we can use the _more advanced, expensive & powerful_ (GmbH with golden share) solution.
> 
> @wlstylr @stephen-rowan How should we decide on how to decide? When voting does everyone has the same voting power?

I went through the process of setting up an LLC on Otoco as a test.  It's an interesting solution, but there are several drawbacks. 

Setting up was quick and easy but you are locked in the ETH ecosystem.  Token distribution, voting, treasury management (Gnosis safe) would all be needed to be done through ETH.  The company would be ETH  native, as would the processes in decisions we made regarding it.  

I'm leaning towards setting up a Verein.

**Comment by DanM3rcurius** [comment]
Yes. We need money for the entity, let's see if we can pull off this drop and then incorporate.
GitHub account is under SecretDecks, we are also officially recognised as an open soruce project by GitBook and are on the free plan. Discord server "belongs" to Discord, and we have 4 admins there . Also since there's no payment flowing to neither GiitHub, GitBook or Discord, the ownership transfer is not really necessary or not as important as the domain and yes newsletter!

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
